src/forms/form-control.scss

```diff
@@ -130,8 +130,7 @@ textarea.form-control {
       // stylelint-disable-next-line primer/spacing
       padding: 2px $spacer-1;
       font-style: normal;
-      // stylelint-disable-next-line primer/colors
-      background: var(--color-auto-yellow-1);
+      background: var(--color-bg-warning);
```

Comment:
Before | After
--- | ---
![Screen Shot 2021-06-07 at 22 10 43](https://user-images.githubusercontent.com/378023/121023487-4a691580-c7de-11eb-9691-77e921d21f03.png) | ![Screen Shot 2021-06-07 at 22 11 01](https://user-images.githubusercontent.com/378023/121023494-4b9a4280-c7de-11eb-8d2c-875a7c71827c.png)

I initially used `var(--color-attention-subtle, var(--color-auto-yellow-1))`, but `--color-attention-subtle` is not yet available and will be replaced at a later time.